### ISSUE TITLE
Add markdown link verification

### ISSUE THREAD

**Issue by hofbi** [rvben/rumdl#633]
### Problem or motivation

So far this hook is a replacement for various markdown linters and formatters. But there is also https://github.com/tcort/markdown-link-check which is a super useful tool as it verifies that all your links used in markdown files are actually correct.

I mainly use this for local/relative links and disable web requests as I don't want to depend on network connectivity. But even for local/relative links I find this incredible useful, in particular for larger projects where you want to make sure that your docs stay up to date.

### Proposed solution

Provide similar functionality that https://github.com/tcort/markdown-link-check or maybe even support only local/relative link verification.

### Feature category

Other

### Example markdown

File `foo.md`
```markdown
I am Foo
```

File `bar.md`
```markdown
My reference to [foo](foo.md)
```

Now if `foo.md` or `bar.md` would be moved to another folder, for instance `tmp/`, https://github.com/tcort/markdown-link-check would tell me that `My reference to [foo](foo.md)` is no longer valid and I would have to update the link.

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by hofbi** [comment]
Looks like this is already supported, https://github.com/rvben/rumdl/blob/main/docs/link-validation.md
